modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -50,7 +50,7 @@ class {{{classname}}} {
   {{/isEnum}}
   {{^isBinary}}
   @JsonKey(
-    nullable: {{#nullable}}true{{/nullable}}{{^nullable}}false{{/nullable}},
+    {{#defaultValue}}defaultValue: {{{defaultValue}}},{{/defaultValue}}{{^defaultValue}}nullable: {{#nullable}}true,{{/nullable}}{{^nullable}}false,{{/nullable}}{{/defaultValue}}
```

Comment:
I noticed a problem earlier today. It's not `{{nullable}}` - it is `{{isNullable}}`.
And then you can shorten this to `nullable: {{isNullable}}`

Reply:
aa that's why it was behaving weird sometimes!

Reply:
Ya found some places in other templates that had wrong output when I tried some NNBD stuff.